swap/protocol_test.go

```diff
@@ -251,6 +251,10 @@ func TestTriggerPaymentThreshold(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	debitorSwap.handleConfirmChequeMsg(ctx, creditor, &ConfirmChequeMsg{
```

Comment:
I would place these additional lines line just above the lines containing:
```golang
if creditor.getBalance() != 0 {
		t.Fatalf("Expected debitorSwap balance to be 0, but is %d", creditor.getBalance())
	}
```

Reply:
Not sure what you mean here. Move which lines where exactly?